Assets/MRTK/SDK/Experimental/PulseShader/Scripts/HideOnDevice.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HideOnDevice : MonoBehaviour
+{
```

Comment:
This script needs a `Microsoft.MixedReality.Toolkit` namespace

Reply:
Added namespace Microsoft.MixedReality.Toolkit.Experimental.SurfacePulse